woocommerce/wc-admin
# Match stock status value in CSV download to the table

Fixes #7283 

This PR replaces the following status values in CSV download to match the value displayed on the table.

instock -> In stock
lowstock -> Low stock
onbackordered  On Backordered
outofstock -> Out of stock

### Detailed test instructions:

**Testing client-side**

1. Clone this branch and run `npm start`
2. Add some products and set stock value.
3. Navigate to Analytics -> Stocks
4. Click the Download button.
5. Open the downloaded file and confirm the status values match the table.

**Testing server-side**

1. Clone this branch and run `npm start`
2. Install [WP Mail Logging](https://wordpress.org/plugins/wp-mail-logging/) plugin
2. Add at least two products with `manage stock?` option checked in the `Inventory` tab.
3. Open `client/report/stock/table.js`. Locate `tableQuery` object. Add `per_page` value. The object should look like below with `per_page` added.

```
tableQuery={ {
	orderby: query.orderby || 'stock_status',
	order: query.order || 'asc',
	type: query.type || 'all',
	per_page: 1,
} }
```

4. Navigate to Analytics -> Stocks
5. Click the Download button.
6. Navigate to Tools -> WP Mail Log and check the recently sent email. 
7. Click the download link in the email and confirm the status value is correct.
